index.html

```diff
@@ -1,14 +1,239 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
+
+<head>
     <title>Mockup Home</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+    <link rel="stylesheet" href="styles.css">

```

Comment:
Reference this for the other pages where I suggest moving your CSS out of a style tag and into your css file